badge_server/templates/one-badge.html

```diff
@@ -108,7 +108,7 @@ <h3 class="title-3 m-b-30">
                                 <i class="zmdi zmdi-account-calendar"></i>Dependency Check Result: <span class="status {{ dependency_res.status.name | lower }}">{{ dependency_res.status.value }}</span>
                             </h3>
                             <div class="table-responsive table-data">
-                                {% if dependency_res.status.name in ['LOW_PRIORITY', 'HIGH_PRIORITY'] %}
+                                {% if dependency_res.status.name == 'OUTDATED_DEPENDENCY' %}
```

Comment:
```suggestion
                                {% if dependency_res.status.name in ['OUTDATED_DEPENDENCY', 'OBSOLETE_DEPENDENCY'] %}
```

Reply:
I thought the new status `OBSOLETE_DEPENDENCY` is the same as deprecated dependencies, is that correct?

Reply:
And the deprecated dependencies is displayed as a list, while the outdated dependency is displayed as a table, so that I let them in two different if-else branches.

Reply:
It's not. Please take a look at this for the old to new mapping:
https://github.com/GoogleCloudPlatform/cloud-opensource-python/blob/70b41cadaf7532cf03cee2e0bcc16f2706c0c5b7/badge_server/main.py#L133

Reply:
Acknowledged, thx!